onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -396,10 +396,15 @@
     <string name="trip_list_context_showroute">Show route</string>
 
     <!--  Trip notifications -->
-    <string name="trip_stat_gone">Route %s has departed.</string>
-    <string name="trip_stat_lessthanone">Route %s is less than a minute away!</string>
-    <string name="trip_stat_one">Route %s is 1 minute away!</string>
-    <string name="trip_stat">Route %1$s is %2$d minutes away!</string>
+    <!--  Trip notifications -->
+    <string name="trip_stat_gone_arrived">Route %s has arrived.</string>
+    <string name="trip_stat_gone_departed">Route %s has departed.</string>
+    <string name="trip_stat_lessthanone_arriving">Route %s is arriving in less than a minute!</string>
+    <string name="trip_stat_lessthanone_departing">Route %s is departing in less than a minute!</string>
+    <string name="trip_stat_one_arriving">Route %s is arriving in 1 minute!</string>
+    <string name="trip_stat_one_departing">Route %s is departing in 1 minute!</string>
+    <string name="trip_stat_arriving">Route %1$s is arriving in %2$d minutes!</string>
+    <string name="trip_stat_departing">Route %1$s is departing in %2$d minutes!</string>
```

Comment:
Let's actually change the following strings to abbreviate minutes to min:

~~~
<string name="trip_stat_arriving">Route %1$s is arriving in %2$d min!</string>
<string name="trip_stat_departing">Route %1$s is departing in %2$d min!</string>
~~~

This should eliminate `trip_stat_one_arriving` and `trip_stat_one_departing` because we don't need singular vs. plural anymore.